$NetBSD: patch-tests_CMakeLists.txt,v 1.3 2026/03/17 19:33:33 bsiegert Exp $

* OpenSSH 10 support
* Search for sshd also in /usr/lib/ssh where it is located
  in SunOS/SmartOS.

--- tests/CMakeLists.txt.orig	2026-02-10 09:47:00.000000000 +0000
+++ tests/CMakeLists.txt
@@ -192,7 +192,8 @@ find_program(SSHD_EXECUTABLE
              PATHS
                 /sbin
                 /usr/sbin
-                /usr/local/sbin)
+                /usr/local/sbin
+                /usr/lib/ssh)
 
 if (WITH_PKCS11_URI)
     find_package(softhsm)
